File: app/src/main/java/com/dominikgaller/alpinebooking/booking/inbound/rest/response/RequestTourBookingResponse.java
package com.dominikgaller.alpinebooking.booking.inbound.rest.response;

/**
 * REST response body for UC01 – RequestTourBooking.
 *
 * <p>SDD: See {@code documentation/use-cases/uc01-request-tour-booking.spec.md}, section 3.
 */
public record RequestTourBookingResponse(
        String bookingId,
        String status
) {
}

File: src/main/kotlin/com/example/contractmanagement/booking/inbound/rest/response/RequestTourBookingResponse.kt
package com.example.contractmanagement.booking.inbound.rest.response

/**
 * REST response body for UC01 — RequestTourBooking.
 *
 * SDD: see `documentation/use-cases/uc01-request-tour-booking.spec.md` § 9.
 */
data class RequestTourBookingResponse(
    val bookingId: String,
    val status: String,
)
